src/components/side-panel/panel-header.js

```diff
@@ -297,7 +297,7 @@ function PanelHeaderFactory(SaveExportDropdown, CloudStorageDropdown) {
                       if (item.dropdownComponent) {
                         showExportDropdown(item.id);
                       }
-                      item.onClick();
+                      item.onClick && item.onClick(this.props);
```

Comment:
Do we actually need to pass all the props, or can we just pick the one(s) we need? I generally favor more careful approach (eventually with companion types).

Reply:
How do we do that? The action can choose to call any action bound prop, which would be hard to predict here.

Reply:
yea for that you will need to add a `getProp` property to the item and have the items decide which prop to pass to `onClick`. performance wise, it's not going to matter. if just for the purpose of expose less props, maybe...but people would still need to know what are the props to choose from though, and they can just say give me all the props

Reply:
Exactly. With regards to `getProp`, they still have to derive it from all props, which would be the same problem as passing `this.props` to `onClick`.